### DISCUSSION TITLE
Enable Preview on channel entries that are closed

### DISCUSSION THREAD

**Discussion by jcogs-design** [ExpressionEngine/ExpressionEngine#3160] [Feature Requests]
A content editor might reasonably wish to preview the rendering of an entry that has been closed: for example if they plan to produce similar content in a new entry etc and wish to reference how the content rendered previously.

However, if you try to preview a channel entry that is set to "Closed" status, the preview template loads but the entry content is missing from it. If you change the status of the entry to "open" (or anything that is 'not closed') the content appears.

Since the change in status doesn't affect the published status of the entry this is a sort-of OK work-around, but is a bit convoluted to explain to a content editor, and they must remember to not then inadvertantly 'save' the entry while previewing it.

It would be better if the Preview function was modified to rendered closed entries. 

I can think of no editorial functional reason why Preview should **not** be able to render such entries.

**Comment by intoeetive** [comment]
@jcogs-design I believe we are allowing closed entries to be displayed in the Live Preview.
Can you tell us:
- what EE version is this
- how does the opening `exp:channel:entries` tag look like in the preview template
- are there any extensions / modules that might affect publishing process

**Reply by intoeetive** [reply]
Upon investigation, found that the status is respected just the way it is in front-end requests.

We'll keep this thread open as changing this would affecting UX a lot, and we can't do this without considering all aspect.

For the time being, please use the solution proposed below:

`status='open{if is_live_preview_request}|closed{/if}'`

**Comment by jcogs-design** [comment]
EE6.4.3 through EE6.4.7
The Channel Entries tag for the template being used for preview (and to display actual page) is
```
{exp:channel:entries channel="news" disable="member_data|pagination" limit="1"}
```
Nothing harmful in add-ons that I can see, copy of installed attached as image.
![Screenshot 2023-03-27 at 08-22-06 Add-Ons ExpressionEngine](https://user-images.githubusercontent.com/13821249/227869399-035bc35f-fb39-4633-8827-697e3d03af68.png)
Let me know if there is anything else that I can let you know about.

**Comment by shbchk** [comment]
Hi everybody! 

Here is what I did to resolve this:


```
{exp:channel:entries 
  channel='films' 
  limit='1' 
  require_entry='yes' 
  status='open{if is_live_preview_request}|closed{/if}'
  disable='member_data|pagination|category_fields' 
  cache='yes' 
  refresh='36000' 
  }
```

**Comment by robinsowell** [comment]
The is live preview conditional is the way I'd prefer to handle this.  I want the page to output exactly like it would on the frontend aside from very, very specific manual instances where I change that.
